bin/crew

```diff
@@ -983,11 +983,54 @@ def prepare_package(destdir)
 
     strip_dir destdir
 
+    # Use patchelf to set need paths for all binaries
+    # currently broken due to glibc issue on armv7l
+    unless ['aarch64', 'armv7l'].include?(ARCH)
+       patchelf_set_need_paths destdir
+    end
+
     # use upx on executables
     shrink_dir destdir
   end
 end
 
+def patchelf_set_need_paths(dir)
+  return if !!@pkg.no_patchelf?
+
+  Dir.chdir dir do
+    puts 'Running patchelf'.lightblue
+    abort('No Patchelf found!').lightred unless File.exist?("#{CREW_PREFIX}/bin/patchelf")
+
+    @execfiles = `find . -executable -type f ! \\( -name \"*.a\" \\) -exec sh -c \"[[ \"\$(head -c4 {})\" == \$'\x7FELF' ]]" \\; -exec ls -1i {} \\; | sort -u -n -s -k1,1 | awk '{print $2}'`.chomp
```

Comment:
Notice that the `[[ str1 == str2 ]]` expression and the `$'\x7F'` ascii escape only works on `bash`.

Reply:
I think `sh` invokes `bash` on ChromeOS, no?

Reply:
When `sh` called, `/usr/local/bin/sh` will be in the first priority of target executable, and the `/usr/local/bin/sh` file is a symlink provided by `bash` package. `/bin/sh` is a symlink to `/bin/dash`

Reply:
![Screenshot_20220130-123120.png](https://user-images.githubusercontent.com/1096701/151710971-13f83b7f-6101-4cc5-b1b4-026a83c046f3.png)

Reply:
@uberhacker looks like we should be using `bash` in lieu of `sh` since `bash` is in neither `core` nor `buildessential`.